errors/error.go

```diff
@@ -0,0 +1,26 @@
+package errors
+
+import "errors"
+
+//Errors for /service/csv.go
+type csvError error
+
+var FileError csvError = errors.New("Couldn't Open Csv File")
+var ColumnParseError csvError = errors.New("First Column of CSV most have an Integer Value")
+var CreateError csvError = errors.New("Couldn't Create the Csv File")
+var BadWrite csvError = errors.New("Couldn't Write into the Csv File")
```

Comment:
A continuous group of vars could be grouped in a var block.

```suggestion
var (
	FileError csvError = errors.New("Couldn't Open Csv File")
	ColumnParseError csvError = errors.New("First Column of CSV most have an Integer Value")
	CreateError csvError = errors.New("Couldn't Create the Csv File")
	BadWrite csvError = errors.New("Couldn't Write into the Csv File")
)
```